src/main/java/org/betonquest/betonquest/api/profile/ProfileProvider.java

```diff
@@ -0,0 +1,47 @@
+package org.betonquest.betonquest.api.profile;
+
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Interface for implementing a profile provider.
+ * Profile providers are responsible for creating {@link Profile}s and
+ * {@link OnlineProfile}s for players.
+ *
+ * A default implementation is available at {@link UUIDProfileProvider}.
```

Comment:
As this is an implementation detail, we dont want to mention the actual implementation here.

Reply:
Removing it